mopidy/mixer.py

```diff
@@ -81,8 +81,50 @@ def trigger_volume_changed(self, volume: int) -> None:
         either because of a call to :meth:`set_volume` or because of any
         external entity changing the volume.
         """
-        logger.debug("Mixer event: volume_changed(volume=%d)", volume)
-        MixerListener.send("volume_changed", volume=volume)
+        logger.debug("Mixer event: volume_changed(volume=%d)", round(self.volume_filter_out(volume)))
+        MixerListener.send("volume_changed", volume=round(self.volume_filter_out(volume)))
```

Comment:
I think the result of `round(self.volume_filter_out(volume))` should be stored in a variable before the logging call, so that it only has to be calculated once.

Reply:
Good point. I correct this in a new commit.